tests/server/models/review-test.js

```diff
@@ -0,0 +1,56 @@
+var sinon = require('sinon');
+var expect = require('chai').expect;
+
+var Sequelize = require('sequelize');
+
+var db = require('../../../server/db');
+
+var Review = db.model('review');
+
+describe('Review model', function () {
+
+    beforeEach('Sync DB', function () {
+        return db.sync({ force: true });
+    });
+
+    it('creates a review',
```

Comment:
This test is a bit unnecessary. You're not testing your code so much as you are testing that Sequelize does what it says it does. I would just skip this test, or make it more detailed with more `expect`s